src/main/java/racing/RacingGame.java

```diff
@@ -0,0 +1,51 @@
+package racing;
+
+import calculator.Calculation;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class RacingGame {
+
+
+    public static void start() {
+        Rule rule = new Rule();
+        rule.setWinnerList();
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).\n");
+        Scanner nameInput = new Scanner(System.in);
+        String[] names = nameInput.next().split(",");
+        Car[] cars = new Car[names.length];
+        for (int idx = 0; idx <= names.length - 1 && rule.isGame; idx++) {
+            cars[idx] = new Car(names[idx]);
+            rule.NameLengthCheck(names[idx]);
+            System.out.println(cars[idx].getName());
+        }
+        inputRound(rule, cars);
+    }
+    public static void inputRound(Rule rule, Car[] cars) {
+        System.out.println("시도할 횟수는 몇 회인가요?.\n");
+        Scanner roundInput = new Scanner(System.in);
+        int round = roundInput.nextInt();
+        rule.setRound(round);
+        System.out.println("실행 결과");
+        roundMatch(rule, cars);
+        rule.finalWinner();
+    }
+    public static void roundMatch(Rule rule, Car[] cars) {
+        for (int roundIdx = 0; roundIdx < rule.round && rule.isGame; roundIdx++) {
+            Run(rule, cars);
+        }
+    }
+    public static void Run(Rule rule, Car[] cars) {
```

Comment:
```suggestion
    public static void run(Rule rule, Car[] cars) {
```

메서드명은 소문자로 시작하는게 좋겠죠?
https://woowacourse.github.io/javable/post/2020-04-26-Method-Naming/